tests/search/selectsummary/complex.sd

```diff
@@ -0,0 +1,51 @@
+# Copyright Vespa.ai. All rights reserved.
+schema complex {
+  document complex {
+    field title type string {
+      indexing: summary | index
+    }
+    field body type string {
+      indexing: index | summary
+    }
+    field a type string {
+      indexing: attribute | summary
+    }
+    field b type string {
+      indexing: attribute | summary
+    }
+    field c type string {
+      indexing: attribute | summary
+    }
+    field d type string {
+      indexing: attribute | summary
+    }
+    field score type int {
+      indexing: attribute
+    }
+  }
+  fieldset default {
+    fields: title, body
+  }
+  document-summary foosum {
```

Comment:
Changing the name of the docsum definitions to be derived from the field names will increase the code readability, e.g `document-summary [a_b|c_d|title_body|title_a_d]`